docs/BasicCommunication/OnSystemRequest/index.md

```diff
@@ -17,34 +17,38 @@ An `OnSystemRequest` is a notification that can be used for a couple of differen
 
 `RequestType` defines the type of the requested data from a mobile device or the cloud. The HMI may request this data, but it's SDL's responsibility to block a `SystemRequest` in case the request intends to transfer a data type not allowed by the policy table.
 
-The HMI is informed about `requestTypes` that are allowed by policies via [OnAppPermissionChanged](../../sdl/onapppermissionchanged) and [OnAppRegistered](../onappregistered).
+HMI is informed about `requestTypes` that are allowed by policies via [OnAppPermissionChanged](../../sdl/onapppermissionchanged) and [OnAppRegistered](../onappregistered).
 
-!!! NOTE
+**_Note_**:   
+  1. If HMI sends `OnSystemRequest` with a request type disallowed by the policy table, SDL will ignore it.   
+  2. If HMI sends 'url' parameter within BasicCommunication.OnSystemRequest, SDL resends the 'url' parameter to mobile app via OnSystemRequest.   
+  3. If HMI(SyncP) doesn't send any URLs to SDL, it is supposed that mobile application will sent Policy Table Update data back to SDL.
 
-If the HMI sends `OnSystemRequest` with a request type disallowed by the policy table, SDL will ignore it.
+_**SyncP Note**_:   
+ 1. It's SyncP responsibility to encrypt and encode PTS file and provide it to SDL via OnSystemRequest HMI API ("filename") parameter.      
+ 2. It's SyncP responsibility to choose an application for sending PTU and start timer (for future retry strategy) after sending OnSystemRequest to SDL.
 
-!!!
+**_HMI must_**:   
+HMI must send `OnSystemRequest`, if specific data is requested from the mobile device/cloud, or binary data needs to be sent to the mobile device.
 
-!!! MUST
-
-HMI must send `OnSystemRequest` if specific data is requested from the mobile device/cloud, or binary data needs to be sent to the mobile device.
-
-!!!
 ### Notification
 
 #### Parameters
 
 |Name|Type|Mandatory|Additional|
 |:---|:---|:--------|:---------|
-|requestType|[Common.RequestType](../../common/enums/#requesttype)|true||
+|requestType|[Common.RequestType]|true||
 |url|String|false|minlength: 1<br>maxlength: 1000|
-|fileType|[Common.FileType](../../common/enums/#filetype)|false||
+|fileType|[Common.FileType]|false||
 |offset|Integer|false|minvalue: 0<br>maxvalue: 100000000000|
 |length|Integer|false|minvalue: 0<br>maxvalue: 100000000000|
 |timeout|Integer|false|minvalue: 0<br>maxvalue: 2000000000|
 |fileName|String|true|minlength: 1<br>maxlength: 255|
 |appID|String|false|minlength: 1<br>maxlength: 50|
 
+[Common.RequestType]: https://github.com/smartdevicelink/sdl_hmi_integration_guidelines/blob/develop/docs/Common/Enums/index.md#requesttype
+[Common.FileType]: https://github.com/smartdevicelink/sdl_hmi_integration_guidelines/blob/develop/docs/Common/Enums/index.md#filetype
```

Comment:
These links should not be made to the GitHub repository directly, they should be relative links like they were originally.